### PR TITLE
clean fetchMarkets

### PR DESCRIPTION
**Pull request by frosty00**
Removed hardcoded elements of fetchMarkets and moved them into describe. 

Removed any places that fetchMarkets extends from hardcoded fees (in preparation for the dynamically loaded fees).

### CODE REVIEW

File: js/_1btcxe.js [hunk 1]
```diff
@@ -53,43 +53,40 @@ module.exports = class _1btcxe extends Exchange {
                     ],
                 },
             },
+            'markets': {
+                'BTC/AED': {'base': 'BTC', 'id': 'AED', 'quote': 'AED', 'symbol': 'BTC/AED'},
```
Comment (kroitor) [inline js/_1btcxe.js]: this will break getbtc...

**Comment by kroitor** [comment]
It would be A LOT easier for me to merge, if you submitted them one by one, because of conflicts (have to merge other requests and bugfixes as well), so it may quickly run out of synch... If you could split it up, that would be awesome. But thx anyway ) Really appreciate your help!

File: js/_1btcxe.js (hunk 1 above)
Comment (frosty00) [reply js/_1btcxe.js]: Ah I see, didn't release it subclass _1btcxe >:

Comment (kroitor) [reply js/_1btcxe.js]: Yep, having them in fetchMarkets is a special case when both the base and the child have them hardcoded...

Comment (kroitor) [reply js/_1btcxe.js]: Haven't come up with a better way to handle it yet.

**Comment by frosty00** [comment]
Is there a command line trick to do this?

Submitting 16 different pull requests is a bit tedious...

**Comment by kroitor** [comment]
@frosty00 don't worry about this one, will merge as soon as it passes the syntax checks, that was mostly for the future – having them separate means merging them faster. To fix this particular pr, we need to add two commas to the independentreserve.js, to avoid for dangling lines in assoc dicts. If you can do that, I'll merge right away. Thanks!

**Pull request merged**